webapp/alembic/versions/52fbe49dfccb_updated_assets_fields.py

```diff
@@ -0,0 +1,60 @@
+"""empty message
+
+Revision ID: 52fbe49dfccb
+Revises: a8efd843e4ad
+Create Date: 2024-09-24 09:50:04.631841
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "52fbe49dfccb"
+down_revision = "a8efd843e4ad"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "product",
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("created", sa.DateTime(), nullable=False),
+        sa.Column("updated", sa.DateTime(), nullable=False),
+        sa.PrimaryKeyConstraint("name"),
+    )
+    op.add_column("asset", sa.Column("asset_type", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("name", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("product", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("author", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("google_drive_link", sa.String(), nullable=True))
+    op.add_column(
+        "asset", sa.Column("salesforce_campaign_id", sa.String(), nullable=True)
+    )
+    op.add_column("asset", sa.Column("language", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("redirect", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("redirect", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("tag", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("tag", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("token", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("token", sa.Column("updated", sa.DateTime(), nullable=False))
```

Comment:
why is there a duplication here?

Reply:
It's actually separate fields, for `created` and `updated` on the `tag` and `token` models. I added them for better record keeping